### ISSUE TITLE
JS API feature request - Frequency analysis

### ISSUE THREAD

**Issue by Jeronimo95** [rdio/api#170]
The JS API is missing a feature of the Web Playback API - frequency analysis. I think this is a fantastic feature and would like to see it added to the JS API.

**Comment by iangilman** [comment]
Agreed, thanks for posting this. This feature is not in progress at the moment, but we'll update here if/when it happens!
